.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--- Provide a short summary of your changes in the title above -->
+
+## Description
+<!--- Describe your changes in detail, what problems does it solve? See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) for tips on writing a good commit message -->
+
+## Related Issues
+<!--- If you are suggesting a new feature or change, please create an issue first -->
+<!--- Add a link to all corresponding Github issues here, using any [appropriate keywords](https://help.github.com/en/articles/closing-issues-using-keywords) as appropriate -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Documentation (non-breaking change that does not add functionality but updates documentation)
+- [ ] Chore (non-breaking change that does not add functionality or fix an issue)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the [**CONTRIBUTING**](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md) document.
+- [ ] Used the same coding conventions as the rest of the project
```

Comment:
This is fine for now, but just getting a rubocop check in travis is probably better.

Reply:
yeah, but until we have that, I'd like to at least have a guideline.